dash/dash-renderer/src/TreeContainer.js

```diff
@@ -252,24 +253,51 @@ class BaseTreeContainer extends Component {
 
         for (let i = 0; i < childrenProps.length; i++) {
             const childrenProp = childrenProps[i];
+
+            const handleObject = (obj, opath) => {
+                return keys(obj).reduce((acc, k) => {
+                    const node = acc[k];
+                    return {
+                        ...acc,
+                        [k]: this.wrapChildrenProp(node, [...opath, k])
+                    };
+                }, obj);
```

Comment:
If I'm understanding this correctly:
```suggestion
                return mapObjIndexed(
                    (node, k) => this.wrapChildrenProp(node, [...opath, k]),
                    obj
                );
```

Reply:
Yes, just to add the import for mapObjIndexed.